### ISSUE TITLE
Minimal sample not working: authentication canceled

### ISSUE THREAD

**Issue by michelleran** [playgameservices/play-games-plugin-for-unity#2863]
Hi! I'm trying to run the minimal sample on my device (Pixel 2, Android 10), but although a banner saying "Connecting to Play Games" shows up, it quickly disappears and returns "Authentication canceled." My dev environment is Unity 2019.3.0f3 on OS X 10.15.1.

Things I've already done:

- copied the resource configuration
- created a keystore
- created a linked app & added the id
- am listed as a tester

I also checked the Android Studio SDK Manager and have the latest version of Google Play Services (though I couldn't find the Android Support Library, Android Support Repository, or Google Repository?)

This is the output from logcat (with `PlayGamesPlatform.DebugLogEnabled`):
```
03-29 21:32:07.007 16060 16081 I Unity   : Starting Auth with token client. 
03-29 21:32:07.007 16060 16081 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
03-29 21:32:07.007 16060 16081 I Unity   : 
03-29 21:32:07.035 16060 16081 I Unity   :  [Play Games Plugin DLL] 03/29/20 21:32:07 -04:00 DEBUG: Activating PlayGamesPlatform. 
03-29 21:32:07.035 16060 16081 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
03-29 21:32:07.035 16060 16081 I Unity   : 
03-29 21:32:07.035 16060 16081 I Unity   :  [Play Games Plugin DLL] 03/29/20 21:32:07 -04:00 DEBUG: Instance was not initialized, using default configuration. 
03-29 21:32:07.035 16060 16081 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
03-29 21:32:07.035 16060 16081 I Unity   : 
03-29 21:32:07.036 16060 16081 W Unity   : !!! [Play Games Plugin DLL] 03/29/20 21:32:07 -04:00 WARNING: Creating new PlayGamesPlatform 
03-29 21:32:07.036 16060 16081 W Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
03-29 21:32:07.036 16060 16081 W Unity   : 
03-29 21:32:07.036 16060 16081 I Unity   :  [Play Games Plugin DLL] 03/29/20 21:32:07 -04:00 DEBUG: PlayGamesPlatform activated: GooglePlayGames.PlayGamesPlatform 
03-29 21:32:07.036 16060 16081 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
03-29 21:32:07.036 16060 16081 I Unity   : 
03-29 21:32:07.036 16060 16081 I Unity   :  [Play Games Plugin DLL] 03/29/20 21:32:07 -04:00 DEBUG: Creating platform-specific Play Games client. 
03-29 21:32:07.036 16060 16081 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
03-29 21:32:07.036 16060 16081 I Unity   : 
03-29 21:32:07.036 16060 16081 I Unity   :  [Play Games Plugin DLL] 03/29/20 21:32:07 -04:00 DEBUG: Creating Android IPlayGamesClient Client 
03-29 21:32:07.036 16060 16081 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
03-29 21:32:07.036 16060 16081 I Unity   : 
03-29 21:32:07.096 16060 16081 D HelperFragment: Creating fragment
03-29 21:32:07.096 16060 16060 D HelperFragment: onResume called
03-29 21:32:07.096 16060 16060 D SignInRequest: signIn
03-29 21:32:07.097 16060 16060 D SignInRequest: Building client for: b0228e2 (a:false e:false i:false wc: 124354490430-goo565nhsukcif2pjtmfu0f9h48f84vb.apps.googleusercontent.com f: false)
03-29 21:32:07.102 16060 16060 D SignInRequest: signInClient.silentSignIn
03-29 21:32:07.174 14291 14331 W GamesServiceBroker: Client connected with SDK 200914000, Services 200914037, and Games 168400048
03-29 21:32:07.192 14291 14321 I PeopleChimeraService: onService. callbacks = aagj@7162a17, request = com.google.android.gms.common.internal.GetServiceRequest@a0d1e04
03-29 21:32:07.202 14291 14331 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:07.210 14291 14331 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:07.216 14291 14359 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:07.219 13541 13541 W ChimeraUtils: Non Chimera context
03-29 21:32:07.219 13541 13541 W ChimeraUtils: Non Chimera context
03-29 21:32:07.230 14291 14331 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:07.232 14291 14359 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:07.253   855   855 I /vendor/bin/hw/android.hardware.health@2.0-service.wahoo: SRAM data: 2779000
03-29 21:32:07.389 24561 24561 I wpa_supplicant: RRM: Ignoring radio measurement request: Not RRM network
03-29 21:32:07.401 14291 14359 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:07.406 16060 16060 D SignInRequest: silentSignIn.onFailure
03-29 21:32:05.972  2643  3278 I chatty  : uid=10022(com.android.systemui) pool-5-thread-1 identical 2 lines
03-29 21:32:07.256  2643  3278 D PowerUI : can't show warning due to - plugged: true status unknown: false
03-29 21:32:07.408  1298  9231 I ActivityTaskManager: START u0 {act=com.google.android.gms.auth.GOOGLE_SIGN_IN pkg=com.mran.cake cmp=com.mran.cake/com.google.android.gms.auth.api.signin.internal.SignInHubActivity (has extras)} from uid 10244
03-29 21:32:07.409   862   862 D android.hardware.power@1.3-service.pixel-libperfmgr: LAUNCH: 1
03-29 21:32:07.411 14291 14331 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:07.447 16060 16081 D         : PlayerBase::stop() from IPlayer
03-29 21:32:07.447 16060 16081 D AudioTrack: stop(5062): called with 257760 frames delivered
03-29 21:32:07.457 16060 16060 W ActivityThread: handleWindowVisibility: no activity for token android.os.BinderProxy@6bd5448
03-29 21:32:07.463  1298  3068 I ActivityTaskManager: START u0 {act=com.google.android.gms.auth.GOOGLE_SIGN_IN pkg=com.google.android.gms cmp=com.google.android.gms/.auth.api.signin.ui.SignInActivity (has extras)} from uid 10244
03-29 21:32:07.473   862   862 D android.hardware.power@1.3-service.pixel-libperfmgr: LAUNCH: 1
03-29 21:32:07.492 10940 10940 W ActivityThread: handleWindowVisibility: no activity for token android.os.BinderProxy@240e80d
03-29 21:32:07.498 16060 16134 I Adreno  : QUALCOMM build                   : bc00834, I609ab310b2
03-29 21:32:07.498 16060 16134 I Adreno  : Build Date                       : 04/11/19
03-29 21:32:07.498 16060 16134 I Adreno  : OpenGL ES Shader Compiler Version: EV031.26.07.00
03-29 21:32:07.498 16060 16134 I Adreno  : Local Branch                     : 
03-29 21:32:07.498 16060 16134 I Adreno  : Remote Branch                    : 
03-29 21:32:07.498 16060 16134 I Adreno  : Remote Branch                    : 
03-29 21:32:07.498 16060 16134 I Adreno  : Reconstruct Branch               : 
03-29 21:32:07.498 16060 16134 I Adreno  : Build Config                     : S L 8.0.6 AArch32
03-29 21:32:07.499 16060 16134 I Adreno  : PFP: 0x005ff110, ME: 0x005ff066
03-29 21:32:07.511   846  1021 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_AUDPROC_GAIN_DEP_STEP_TABLE, vol index 0
03-29 21:32:07.511   846  1021 E volume_listener: check_and_set_gain_dep_cal: Failed to set gain dep cal level
03-29 21:32:07.512   846  1021 E volume_listener: check_and_set_gain_dep_cal: Failed to set gain dep cal level
03-29 21:32:07.519   793 16140 E ResolverController: No valid NAT64 prefix (1679, <unspecified>/0)
03-29 21:32:07.548  2643  2643 I Elmyra/ElmyraService: Unblocked; current action: LaunchOpa [mIsGestureEnabled -> true; mIsOpaEnabled -> true]
03-29 21:32:07.549   850   850 V ContextHubHal: sendMessageToHub
03-29 21:32:07.549   937   984 D CHRE    : @ 3473863.872: Parsed nanoapp message from host: app ID 0x476f6f676c00100e, endpoint 0x1, msgType 200, payload size 10
03-29 21:32:07.550   937   984 D CHRE    : @ 3473863.873: Elmyra/Nanoapp Unexpected event: 512
03-29 21:32:07.552 14291 14548 W GamesServiceBroker: Client connected with SDK 200914000, Services 200914037, and Games 168400048
03-29 21:32:07.556   862   862 D android.hardware.power@1.3-service.pixel-libperfmgr: LAUNCH: 0
03-29 21:32:07.559  1298  1346 I ActivityTaskManager: Displayed com.google.android.gms/.auth.api.signin.ui.SignInActivity: +61ms
03-29 21:32:07.561 14291 14331 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:07.568 14291 14359 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:07.573   937   984 W CHRE    : @ 3473863.896: Incorrect number of SMGR samples for vendor0: 1
03-29 21:32:07.573   937   984 W CHRE    : @ 3473863.897: Dropping zero readingCount event
03-29 21:32:07.576 13541 13541 W ChimeraUtils: Non Chimera context
03-29 21:32:07.577 13541 13541 W ChimeraUtils: Non Chimera context
03-29 21:32:07.580 14291 14359 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:07.593   937   984 W CHRE    : @ 3473863.916: Incorrect number of SMGR samples for vendor0: 3
03-29 21:32:07.593   937   984 W CHRE    : @ 3473863.916: Incorrect vendor0 SMGR sample sequence
03-29 21:32:07.593   937   984 W CHRE    : @ 3473863.916: Incorrect vendor0 SMGR sample sequence
03-29 21:32:07.594 14291 14359 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:07.596 14291 14331 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:07.639 13541 13541 W ChimeraUtils: Non Chimera context
03-29 21:32:07.639 13541 13541 W ChimeraUtils: Non Chimera context
03-29 21:32:07.722 14291 13921 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:07.726 14291 14359 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:07.733  1298  3051 I ActivityTaskManager: START u0 {cmp=com.google.android.gms/.common.api.GoogleApiActivity (has extras)} from uid 10024
03-29 21:32:07.734   862   862 D android.hardware.power@1.3-service.pixel-libperfmgr: LAUNCH: 1
03-29 21:32:07.749 10940 10940 W ActivityThread: handleWindowVisibility: no activity for token android.os.BinderProxy@f67f4ce
03-29 21:32:07.759  1298  5509 I ActivityTaskManager: START u0 {act=com.google.android.gms.games.SIGN_IN flg=0x4000000 pkg=com.google.android.play.games cmp=com.google.android.play.games/com.google.android.gms.games.ui.signin.SignInActivity (has extras)} from uid 10024
03-29 21:32:07.761   862   862 D android.hardware.power@1.3-service.pixel-libperfmgr: LAUNCH: 1
03-29 21:32:07.770 14291 16130 W FA-SVC  : Interrupted in onRunTask while uploading
03-29 21:32:07.781 13388 13388 W ActivityThread: handleWindowVisibility: no activity for token android.os.BinderProxy@ba09f28
03-29 21:32:07.897  1298  1346 I ActivityTaskManager: Displayed com.google.android.play.games/com.google.android.gms.games.ui.signin.SignInActivity: +47ms
03-29 21:32:07.905 13388 13388 E ThemeUtils: View class com.airbnb.lottie.LottieAnimationView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
03-29 21:32:07.909 13388 13388 I chatty  : uid=10149(com.google.android.play.games) identical 3 lines
03-29 21:32:07.910 13388 13388 E ThemeUtils: View class com.airbnb.lottie.LottieAnimationView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
03-29 21:32:07.927 13541 16129 E NetworkScheduler.ATC: Called cancelTask for already completed task com.google.android.gms/.measurement.PackageMeasurementTaskService{u=0 tag="Measurement.PackageMeasurementTaskService.UPLOAD_TASK_TAG" trigger=window{start=705s,end=1412s,earliest=-14s,latest=692s} requirements=[NET_CONNECTED] attributes=[PERSISTED] scheduled=-719s last_run=0s jid=N/A status=ACTIVE retries=0 client_lib=GMS_TASK_SCHEDULER-200914000} :1
03-29 21:32:07.940 14291 14548 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:07.941   862   862 D android.hardware.power@1.3-service.pixel-libperfmgr: LAUNCH: 0
03-29 21:32:08.019 13541 16133 E NetworkScheduler.ATC: Called cancelTask for already completed task com.google.android.gms/.measurement.PackageMeasurementTaskService{u=0 tag="Measurement.PackageMeasurementTaskService.UPLOAD_TASK_TAG" trigger=window{start=705s,end=1412s,earliest=-14s,latest=692s} requirements=[NET_CONNECTED] attributes=[PERSISTED] scheduled=-719s last_run=0s jid=N/A status=ACTIVE retries=0 client_lib=GMS_TASK_SCHEDULER-200914000} :1
03-29 21:32:08.037 13541 16129 E NetworkScheduler.ATC: Called cancelTask for already completed task com.google.android.gms/.measurement.PackageMeasurementTaskService{u=0 tag="Measurement.PackageMeasurementTaskService.UPLOAD_TASK_TAG" trigger=window{start=705s,end=1412s,earliest=-14s,latest=692s} requirements=[NET_CONNECTED] attributes=[PERSISTED] scheduled=-719s last_run=0s jid=N/A status=ACTIVE retries=0 client_lib=GMS_TASK_SCHEDULER-200914000} :1
03-29 21:32:08.114 13541 16133 E NetworkScheduler.ATC: Called cancelTask for already completed task com.google.android.gms/.measurement.PackageMeasurementTaskService{u=0 tag="Measurement.PackageMeasurementTaskService.UPLOAD_TASK_TAG" trigger=window{start=705s,end=1412s,earliest=-15s,latest=691s} requirements=[NET_CONNECTED] attributes=[PERSISTED] scheduled=-720s last_run=0s jid=N/A status=ACTIVE retries=0 client_lib=GMS_TASK_SCHEDULER-200914000} :1
03-29 21:32:08.130 13541 16133 E NetworkScheduler.ATC: Called cancelTask for already completed task com.google.android.gms/.measurement.PackageMeasurementTaskService{u=0 tag="Measurement.PackageMeasurementTaskService.UPLOAD_TASK_TAG" trigger=window{start=705s,end=1412s,earliest=-15s,latest=691s} requirements=[NET_CONNECTED] attributes=[PERSISTED] scheduled=-720s last_run=0s jid=N/A status=ACTIVE retries=0 client_lib=GMS_TASK_SCHEDULER-200914000} :1
03-29 21:32:09.166   855   855 I /vendor/bin/hw/android.hardware.health@2.0-service.wahoo: SRAM data: 2779000
03-29 21:32:09.178  2643  3278 D PowerUI : can't show warning due to - plugged: true status unknown: false
03-29 21:32:09.581 13541 16129 E NetworkScheduler.ATC: Called cancelTask for already completed task com.google.android.gms/.measurement.PackageMeasurementTaskService{u=0 tag="Measurement.PackageMeasurementTaskService.UPLOAD_TASK_TAG" trigger=window{start=705s,end=1412s,earliest=-16s,latest=690s} requirements=[NET_CONNECTED] attributes=[PERSISTED] scheduled=-721s last_run=-2s jid=N/A status=ACTIVE retries=0 client_lib=GMS_TASK_SCHEDULER-200914000} :1
03-29 21:32:09.672 14291 16160 I Authzen : [DeviceStateSyncManager] The server is in sync with current state. Nothing to do
03-29 21:32:10.534   846  1021 D audio_hw_primary: disable_audio_route: usecase(5) reset and update mixer path: audio-ull-playback speaker
03-29 21:32:10.534   862   862 D android.hardware.power@1.3-service.pixel-libperfmgr: AUDIO_LOW_LATENCY: 0
03-29 21:32:10.536   846  1021 D audio_hw_primary: disable_snd_device: snd_device(2: speaker)
03-29 21:32:10.971 13541 13541 I BeaconBle: 'L' hardware scan: scan stopped, no clients
03-29 21:32:11.065 14291 14548 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:11.066 14291 14453 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:11.067 13541 13541 W ChimeraUtils: Non Chimera context
03-29 21:32:11.067 13541 13541 W ChimeraUtils: Non Chimera context
03-29 21:32:11.069 13388 13453 W PlayCommon: [80408] osi.a: No account for auth token provided
03-29 21:32:11.073   793 16162 E ResolverController: No valid NAT64 prefix (1679, <unspecified>/0)
03-29 21:32:11.090 14291 14548 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:11.094 13541 13541 W ChimeraUtils: Non Chimera context
03-29 21:32:11.094 13541 13541 W ChimeraUtils: Non Chimera context
03-29 21:32:11.118   793 16164 E ResolverController: No valid NAT64 prefix (1679, <unspecified>/0)
03-29 21:32:11.138 14291 14428 W Conscrypt: Could not set socket write timeout: java.net.SocketException: Socket closed
03-29 21:32:11.139 14291 14428 W Conscrypt: 	at com.google.android.gms.org.conscrypt.Platform.setSocketWriteTimeout(:com.google.android.gms@200914037@20.09.14 (120400-300565878):2)
03-29 21:32:11.139 14291 14428 W Conscrypt: 	at com.google.android.gms.org.conscrypt.ConscryptFileDescriptorSocket.setSoWriteTimeout(:com.google.android.gms@200914037@20.09.14 (120400-300565878):0)
03-29 21:32:11.166 14291 14428 W Conscrypt: Could not set socket write timeout: java.net.SocketException: Socket closed
03-29 21:32:11.167 14291 14428 W Conscrypt: 	at com.google.android.gms.org.conscrypt.Platform.setSocketWriteTimeout(:com.google.android.gms@200914037@20.09.14 (120400-300565878):2)
03-29 21:32:11.167 14291 14428 W Conscrypt: 	at com.google.android.gms.org.conscrypt.ConscryptFileDescriptorSocket.setSoWriteTimeout(:com.google.android.gms@200914037@20.09.14 (120400-300565878):0)
03-29 21:32:11.352 14291 14453 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:11.352 14291 14548 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:11.464 13388 13388 W SignInActivity: onSignInFailed()...
03-29 21:32:11.465 13388 13388 W SignInActivity: Sign in failed during 6
03-29 21:32:11.465 13388 13388 W SignInActivity: ==> Returning non-OK result: 10002
03-29 21:32:11.503 14291 14548 W GamesServiceBroker: Client connected with SDK 201000000, Services 200914037, and Games 168400048
03-29 21:32:11.549 10940 10940 W AutoManageHelper: Unresolved error while connecting client. Stopping auto-manage.
03-29 21:32:11.578 13541 16132 E NetworkScheduler.ATC: Called cancelTask for already completed task com.google.android.gms/.measurement.PackageMeasurementTaskService{u=0 tag="Measurement.PackageMeasurementTaskService.UPLOAD_TASK_TAG" trigger=window{start=705s,end=1412s,earliest=-18s,latest=688s} requirements=[NET_CONNECTED] attributes=[PERSISTED] scheduled=-723s last_run=-4s jid=N/A status=ACTIVE retries=0 client_lib=GMS_TASK_SCHEDULER-200914000} :1
03-29 21:32:11.589  1298  3051 I WindowManager: Queue is too deep! Purged item with taskid=22817
03-29 21:32:11.594   855   855 I /vendor/bin/hw/android.hardware.health@2.0-service.wahoo: SRAM data: 2779000
03-29 21:32:11.609  2643  3278 D PowerUI : can't show warning due to - plugged: true status unknown: false
03-29 21:32:11.610 16060 16060 E SignInRequest: Setting result error status code to: 16
03-29 21:32:11.627  2643  2643 I Elmyra/ElmyraService: Gated by NavigationBarVisibility [mIsNavigationHidden -> true; mExceptions -> [DismissTimer [mReceiverRegistered -> true], SnoozeAlarm [mReceiverRegistered -> true], SilenceCall [mSilenceSettingEnabled -> true], SettingsAction]; mIsNavigationGestural -> false; isActiveAssistantNga() -> false]
03-29 21:32:11.627   850   850 V ContextHubHal: sendMessageToHub
03-29 21:32:11.628   937   984 D CHRE    : @ 3473867.951: Parsed nanoapp message from host: app ID 0x476f6f676c00100e, endpoint 0x1, msgType 201, payload size 0
03-29 21:32:11.652 16060 16060 I Unity   : Returning an error code. 
03-29 21:32:11.652 16060 16060 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
03-29 21:32:11.652 16060 16060 I Unity   : 
03-29 21:32:11.661 16060 16060 D HelperFragment: onResume called
03-29 21:32:11.686 16060 16060 V MediaRouter: Selecting route: RouteInfo{ name=Phone, description=null, status=null, category=RouteCategory{ name=System types=ROUTE_TYPE_LIVE_AUDIO ROUTE_TYPE_LIVE_VIDEO  groupable=false }, supportedTypes=ROUTE_TYPE_LIVE_AUDIO ROUTE_TYPE_LIVE_VIDEO , presentationDisplay=null }
03-29 21:32:11.688   846 26542 D audio_hw_primary: out_set_parameters: enter: usecase(5: audio-ull-playback) kvpairs: routing=2
03-29 21:32:11.689   846 26542 E volume_listener: check_and_set_gain_dep_cal: Failed to set gain dep cal level
03-29 21:32:11.690   846 16607 D audio_hw_primary: enable_snd_device: snd_device(2: speaker)
03-29 21:32:11.690   862   862 D android.hardware.power@1.3-service.pixel-libperfmgr: AUDIO_STREAMING: 1
03-29 21:32:11.690   846 16607 D audio_route: Apply path: speaker
03-29 21:32:11.690   846 16607 D ACDB-LOADER: ACDB -> send_audio_cal, acdb_id = 15, path = 0, app id = 0x11130, sample rate = 48000
03-29 21:32:11.690   846 16607 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_AUDPROC_GAIN_DEP_STEP_TABLE, vol index 0
03-29 21:32:11.690   846 16607 D ACDB-LOADER: ACDB -> GET_AFE_TOPOLOGY_ID for adcd_id 15, Topology Id 112fc
03-29 21:32:11.690   846 16607 D         : Failed to fetch the lookup information of the device 0000000F 
03-29 21:32:11.690   846 16607 D ACDB-LOADER: Error: ACDB AFE returned = -19
03-29 21:32:11.690   846 16607 D audio_hw_primary: enable_audio_route: usecase(5) apply and update mixer path: audio-ull-playback speaker
03-29 21:32:11.690   846 16607 D audio_route: Apply path: audio-ull-playback speaker
03-29 21:32:11.693  1298  3051 D ConnectivityService: requestNetwork for uid/pid:10244/16060 NetworkRequest [ TRACK_DEFAULT id=17669, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED Uid: 10244] ]
03-29 21:32:11.694  1298  2604 D Ethernet: got request NetworkRequest [ TRACK_DEFAULT id=17669, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED Uid: 10244] ] with score 60 and serial -1
03-29 21:32:11.694  1298  2603 D WIFI_AWARE_FACTORY: got request NetworkRequest [ TRACK_DEFAULT id=17669, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED Uid: 10244] ] with score 60 and serial -1
03-29 21:32:11.694  1298  2574 D UntrustedWifiNetworkFactory: got request NetworkRequest [ TRACK_DEFAULT id=17669, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED Uid: 10244] ] with score 60 and serial -1
03-29 21:32:11.694  1298  2574 D WifiNetworkFactory: got request NetworkRequest [ TRACK_DEFAULT id=17669, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED Uid: 10244] ] with score 60 and serial -1
03-29 21:32:11.695  2997  2997 D PhoneSwitcherNetworkRequstListener: got request NetworkRequest [ TRACK_DEFAULT id=17669, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED Uid: 10244] ] with score 60 and serial -1
03-29 21:32:11.697  2643  2643 V KeyguardUpdateMonitor: onSubscriptionInfoChanged()
03-29 21:32:11.698  2643  2643 V KeyguardUpdateMonitor: SubInfo:{id=1, iccId=890141032[****] simSlotIndex=0 carrierId=1187 displayName=AT&T carrierName=AT&T nameSource=0 iconTint=-16746133 mNumber=[****] dataRoaming=0 iconBitmap=android.graphics.Bitmap@a2b3182 mcc 310 mnc 410mCountryIso=us isEmbedded false nativeAccessRules null cardString=890141032[****] cardId=-1 isOpportunistic false mGroupUUID=null mIsGroupDisabled=false profileClass=-1 ehplmns = [] hplmns = [310410, 310410, 310410] subscriptionType=0 mGroupOwner=null carrierConfigAccessRules=null}
03-29 21:32:11.700 16060 16081 I Unity   :  [Play Games Plugin DLL] 03/29/20 21:32:11 -04:00 DEBUG: Invoking user callback on game thread 
03-29 21:32:11.700 16060 16081 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
03-29 21:32:11.700 16060 16081 I Unity   : 
03-29 21:32:11.700 16060 16081 I Unity   :  [Play Games Plugin DLL] 03/29/20 21:32:11 -04:00 DEBUG: Authentication canceled 
03-29 21:32:11.700 16060 16081 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
03-29 21:32:11.700 16060 16081 I Unity   : 
```

Any pointers as to what I'm missing would be appreciated!

**Comment by olehkuznetsov** [comment]
Check if you setup Auth Screen in cloud console, please. Also double check SHA1 key for your apk and the linked app.

**Comment by vinilka8** [comment]
Hi @michelleran and @olehkuznetsov 

This is exactly the same problem I have, I don't need to create a new issue.
What is the issue with it, do you know? 

Thanks in advance

**Comment by StRandom** [comment]
Hi

This is very similar logcat and problem I have. But my example is not minimal sample and I've already created issue - https://github.com/playgameservices/play-games-plugin-for-unity/issues/2874. It sometimes log in OK and next time - the same problem (for the same account). SHA-1 and everything is set up correctly. Any progress with it?

Thanks in advance

**Comment by NicolasVidal** [comment]
Here we are experiencing this issue with 0.10.06 on a new game ... On an old one, everything is ok.

**Comment by NicolasVidal** [comment]
Ok I found the issue, My OAuth Autorisation Screen was not configured in the Google API Console for this project.

**Comment by mpartel** [comment]
^ should probably be put on https://developers.google.com/games/services/android/troubleshooting

Also, it didn't fix the issue for me, and I've tried everything else I can find and think of.
Does the OAuth page need to be verified or request some specific scope or domain?

**Comment by BenDevelopment** [comment]
Isn't this only for web authentication?
I have the same error (Authentication canceled). It worked for several weeks, and now it no longer works without having made any modification.
Is there any problems with Google? Am I the only one?

**Comment by BenDevelopment** [comment]
I've solved my problem:
https://github.com/playgameservices/play-games-plugin-for-unity/issues/2878#issuecomment-641140930
